packages/frontend/src/pages/ParkingSpace/components/ParkingSpaceInfo.tsx

```diff
@@ -13,78 +15,109 @@ export const ParkingSpaceInfo = (props: { listingId: string }) => {
     currency: 'SEK',
   })
 
+  const createOffer = useCreateOffer()
+
+  const onCreateOffer = () => {
+    createOffer.mutate(props, {})
+  }
+
+  const renderStartOfferProcessButton = (listingStatus: ListingStatus) => {
+    if (listingStatus == ListingStatus.Expired) {
```

Comment:
Måste den vara expired? Hur blir den expired?

Reply:
Den blir expired av det schemalagda jobb som finns i leasing. POST offer endpointen i core tillåter heller inte att man skapar offers på listings som inte är expired

Reply:
Cool. Merge it